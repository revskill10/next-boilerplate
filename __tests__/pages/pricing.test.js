/* eslint-env jest */

//import { shallow } from 'enzyme'
import App from '../../pages/pricing.js'
import mockPage from '../../test/mockPage'

/*
describe('With Enzyme', () => {
  it('App shows "Hello world!"', () => {
    const app = shallow(<App />)

    expect(app.find('p').text()).toEqual('Hello World!')
  })
})
*/
describe('With Snapshot Testing', () => {
  it('App shows "Pricing"', () => {
    const component = mockPage(App)
    expect(component).toMatchSnapshot()
  })
})
