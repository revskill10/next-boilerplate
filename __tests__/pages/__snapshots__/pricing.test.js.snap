// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Pricing" 1`] = `
Array [
  <header
    className="MuiPaper-root-28 MuiPaper-elevation4-34 MuiAppBar-root-19 MuiAppBar-positionStatic-23 MuiAppBar-colorDefault-25 Component-appBar-10"
  >
    <div
      className="MuiToolbar-root-55 MuiToolbar-regular-57 MuiToolbar-gutters-56"
    >
      <h2
        className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorInherit-79 MuiTypography-noWrap-76 Component-toolbarTitle-11"
      >
        <a
          href="/"
          onClick={[Function]}
          style={
            Object {
              "color": "black",
              "textDecoration": "none",
            }
          }
        >
          TF
        </a>
      </h2>
      <h2
        className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorInherit-79 MuiTypography-noWrap-76 Component-toolbarTitle-11"
      >
        Disconnected
      </h2>
      <button
        className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-text-87 MuiButton-flat-90"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-86"
        >
          <a
            className="Component-appBarLink-18"
            href="/dashboard"
            onClick={[Function]}
          >
            dashboard_menu
          </a>
        </span>
        <span
          className="MuiTouchRipple-root-114"
        />
      </button>
      <button
        className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-text-87 MuiButton-flat-90"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-86"
        >
          <a
            className="Component-appBarLink-18"
            href="/about"
            onClick={[Function]}
          >
            about_menu
          </a>
        </span>
        <span
          className="MuiTouchRipple-root-114"
        />
      </button>
      <button
        className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-text-87 MuiButton-flat-90"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-86"
        >
          <a
            className="Component-appBarLink-18"
            href="/pricing"
            onClick={[Function]}
          >
            pricing_menu
          </a>
        </span>
        <span
          className="MuiTouchRipple-root-114"
        />
      </button>
      <button
        className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-text-87 MuiButton-flat-90"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-86"
        >
          <a
            className="Component-appBarLink-18"
            href="/contact"
            onClick={[Function]}
          >
            contact_menu
          </a>
        </span>
        <span
          className="MuiTouchRipple-root-114"
        />
      </button>
      <button
        className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-text-87 MuiButton-textPrimary-88 MuiButton-outlined-93 MuiButton-outlinedPrimary-94"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-86"
        >
          Login
        </span>
        <span
          className="MuiTouchRipple-root-114"
        />
      </button>
    </div>
  </header>,
  <main>
    <div
      className="AutoGrid-root-121"
    >
      <div
        className="MuiGrid-container-122 MuiGrid-spacing-xs-24-146"
      >
        <div
          className="MuiGrid-item-123 MuiGrid-grid-xs-1-151"
        />
        <div
          className="MuiGrid-item-123 MuiGrid-grid-xs-9-159"
        >
          <div
            className="Page-heroContent-4"
          >
            <h1
              className="MuiTypography-root-59 MuiTypography-display3-61 MuiTypography-colorTextPrimary-82 MuiTypography-gutterBottom-77 MuiTypography-alignCenter-73"
            >
              Pricing
            </h1>
            <p
              className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextSecondary-83 MuiTypography-alignCenter-73"
            >
              Quickly build an effective pricing table for your potential customers with this layout. It's built with default Material-UI components with little customization.
            </p>
          </div>
          <div
            className="MuiGrid-container-122 MuiGrid-spacing-xs-40-148 MuiGrid-align-items-xs-flex-end-132"
          >
            <div
              className="MuiGrid-item-123 MuiGrid-grid-xs-12-162 MuiGrid-grid-sm-6-170 MuiGrid-grid-md-4-182"
            >
              <div
                className="MuiPaper-root-28 MuiPaper-elevation1-31 MuiPaper-rounded-29 MuiCard-root-219"
              >
                <div
                  className="MuiCardHeader-root-220 Page-cardHeader-5"
                >
                  <div
                    className="MuiCardHeader-content-223"
                  >
                    <span
                      className="MuiTypography-root-59 MuiTypography-headline-64 MuiTypography-alignCenter-73 MuiCardHeader-title-224"
                    >
                      Free
                    </span>
                  </div>
                </div>
                <div
                  className="MuiCardContent-root-226"
                >
                  <div
                    className="Page-cardPricing-6"
                  >
                    <h2
                      className="MuiTypography-root-59 MuiTypography-display2-62 MuiTypography-colorTextPrimary-82"
                    >
                      $
                      0
                    </h2>
                    <h2
                      className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextSecondary-83"
                    >
                      /mo
                    </h2>
                  </div>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    10 users included
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    2 GB of storage
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    Help center access
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    Email support
                  </h3>
                </div>
                <div
                  className="MuiCardActions-root-227 Page-cardActions-7"
                >
                  <button
                    className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-text-87 MuiButton-textPrimary-88 MuiButton-outlined-93 MuiButton-outlinedPrimary-94 MuiButton-fullWidth-110 MuiCardActions-action-228"
                    disabled={false}
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex="0"
                    type="button"
                  >
                    <span
                      className="MuiButton-label-86"
                    >
                      Sign up for free
                    </span>
                    <span
                      className="MuiTouchRipple-root-114"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-item-123 MuiGrid-grid-xs-12-162 MuiGrid-grid-sm-6-170 MuiGrid-grid-md-4-182"
            >
              <div
                className="MuiPaper-root-28 MuiPaper-elevation1-31 MuiPaper-rounded-29 MuiCard-root-219"
              >
                <div
                  className="MuiCardHeader-root-220 Page-cardHeader-5"
                >
                  <div
                    className="MuiCardHeader-content-223"
                  >
                    <span
                      className="MuiTypography-root-59 MuiTypography-headline-64 MuiTypography-alignCenter-73 MuiCardHeader-title-224"
                    >
                      Pro
                    </span>
                    <span
                      className="MuiTypography-root-59 MuiTypography-body1-68 MuiTypography-colorTextSecondary-83 MuiTypography-alignCenter-73 MuiCardHeader-subheader-225"
                    >
                      Most popular
                    </span>
                  </div>
                  <div
                    className="MuiCardHeader-action-222"
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-229"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                      />
                      <path
                        d="M0 0h24v24H0z"
                        fill="none"
                      />
                    </svg>
                  </div>
                </div>
                <div
                  className="MuiCardContent-root-226"
                >
                  <div
                    className="Page-cardPricing-6"
                  >
                    <h2
                      className="MuiTypography-root-59 MuiTypography-display2-62 MuiTypography-colorTextPrimary-82"
                    >
                      $
                      15
                    </h2>
                    <h2
                      className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextSecondary-83"
                    >
                      /mo
                    </h2>
                  </div>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    20 users included
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    10 GB of storage
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    Help center access
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    Priority email support
                  </h3>
                </div>
                <div
                  className="MuiCardActions-root-227 Page-cardActions-7"
                >
                  <button
                    className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-contained-96 MuiButton-containedPrimary-97 MuiButton-raised-99 MuiButton-raisedPrimary-100 MuiButton-fullWidth-110 MuiCardActions-action-228"
                    disabled={false}
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex="0"
                    type="button"
                  >
                    <span
                      className="MuiButton-label-86"
                    >
                      Get started
                    </span>
                    <span
                      className="MuiTouchRipple-root-114"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-item-123 MuiGrid-grid-xs-12-162 MuiGrid-grid-sm-12-176 MuiGrid-grid-md-4-182"
            >
              <div
                className="MuiPaper-root-28 MuiPaper-elevation1-31 MuiPaper-rounded-29 MuiCard-root-219"
              >
                <div
                  className="MuiCardHeader-root-220 Page-cardHeader-5"
                >
                  <div
                    className="MuiCardHeader-content-223"
                  >
                    <span
                      className="MuiTypography-root-59 MuiTypography-headline-64 MuiTypography-alignCenter-73 MuiCardHeader-title-224"
                    >
                      Enterprise
                    </span>
                  </div>
                </div>
                <div
                  className="MuiCardContent-root-226"
                >
                  <div
                    className="Page-cardPricing-6"
                  >
                    <h2
                      className="MuiTypography-root-59 MuiTypography-display2-62 MuiTypography-colorTextPrimary-82"
                    >
                      $
                      30
                    </h2>
                    <h2
                      className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextSecondary-83"
                    >
                      /mo
                    </h2>
                  </div>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    50 users included
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    30 GB of storage
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    Help center access
                  </h3>
                  <h3
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-alignCenter-73"
                  >
                    Phone & email support
                  </h3>
                </div>
                <div
                  className="MuiCardActions-root-227 Page-cardActions-7"
                >
                  <button
                    className="MuiButtonBase-root-111 MuiButton-root-85 MuiButton-text-87 MuiButton-textPrimary-88 MuiButton-outlined-93 MuiButton-outlinedPrimary-94 MuiButton-fullWidth-110 MuiCardActions-action-228"
                    disabled={false}
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex="0"
                    type="button"
                  >
                    <span
                      className="MuiButton-label-86"
                    >
                      Contact us
                    </span>
                    <span
                      className="MuiTouchRipple-root-114"
                    />
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-item-123 MuiGrid-grid-xs-true-150"
        >
          <div
            className="NestedList-root-238"
          >
            <nav
              className="MuiList-root-240 MuiList-padding-241 MuiList-subheader-243"
            >
              <div
                className="MuiListSubheader-root-244 MuiListSubheader-sticky-249 MuiListSubheader-gutters-247"
              >
                Right
              </div>
              <div
                className="MuiButtonBase-root-111 MuiListItem-root-250 MuiListItem-default-253 MuiListItem-gutters-257 MuiListItem-button-258"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex="0"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-229 MuiListItemIcon-root-261"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M2.01 21L23 12 2.01 3 2 10l15 2-15 2z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
                <div
                  className="MuiListItemText-root-262 MuiListItemText-inset-263"
                >
                  <span
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiListItemText-primary-265"
                  >
                    Sent mail
                  </span>
                </div>
                <span
                  className="MuiTouchRipple-root-114"
                />
              </div>
              <div
                className="MuiButtonBase-root-111 MuiListItem-root-250 MuiListItem-default-253 MuiListItem-gutters-257 MuiListItem-button-258"
                onBlur={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex="0"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-229 MuiListItemIcon-root-261"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M21.99 8c0-.72-.37-1.35-.94-1.7L12 1 2.95 6.3C2.38 6.65 2 7.28 2 8v10c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2l-.01-10zM12 13L3.74 7.84 12 3l8.26 4.84L12 13z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
                <div
                  className="MuiListItemText-root-262 MuiListItemText-inset-263"
                >
                  <span
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiListItemText-primary-265"
                  >
                    Drafts
                  </span>
                </div>
                <span
                  className="MuiTouchRipple-root-114"
                />
              </div>
              <div
                className="MuiButtonBase-root-111 MuiListItem-root-250 MuiListItem-default-253 MuiListItem-gutters-257 MuiListItem-button-258"
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex="0"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-229 MuiListItemIcon-root-261"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H4.99c-1.11 0-1.98.9-1.98 2L3 19c0 1.1.88 2 1.99 2H19c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 12h-4c0 1.66-1.35 3-3 3s-3-1.34-3-3H4.99V5H19v10zm-3-5h-2V7h-4v3H8l4 4 4-4z"
                  />
                  <path
                    d="M0 0h24v24H0V0z"
                    fill="none"
                  />
                </svg>
                <div
                  className="MuiListItemText-root-262 MuiListItemText-inset-263"
                >
                  <span
                    className="MuiTypography-root-59 MuiTypography-subheading-66 MuiListItemText-primary-265"
                  >
                    Inbox
                  </span>
                </div>
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-229"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
                <span
                  className="MuiTouchRipple-root-114"
                />
              </div>
              <div
                className="MuiCollapse-container-268 MuiCollapse-entered-269"
                style={
                  Object {
                    "minHeight": "0px",
                  }
                }
              >
                <div
                  className="MuiCollapse-wrapper-270"
                >
                  <div
                    className="MuiCollapse-wrapperInner-271"
                  >
                    <div
                      className="MuiList-root-240"
                    >
                      <div
                        className="MuiButtonBase-root-111 MuiListItem-root-250 MuiListItem-default-253 MuiListItem-gutters-257 MuiListItem-button-258 NestedList-nested-239"
                        onBlur={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex="0"
                      >
                        <svg
                          aria-hidden="true"
                          className="MuiSvgIcon-root-229 MuiListItemIcon-root-261"
                          focusable="false"
                          role="presentation"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
                          />
                          <path
                            d="M0 0h24v24H0z"
                            fill="none"
                          />
                        </svg>
                        <div
                          className="MuiListItemText-root-262 MuiListItemText-inset-263"
                        >
                          <span
                            className="MuiTypography-root-59 MuiTypography-subheading-66 MuiListItemText-primary-265"
                          >
                            Starred
                          </span>
                        </div>
                        <span
                          className="MuiTouchRipple-root-114"
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </nav>
          </div>
        </div>
      </div>
    </div>
  </main>,
  <footer
    className="Component-footer-279 Component-layout-274"
  >
    <div
      className="MuiGrid-container-122 MuiGrid-spacing-xs-32-147 MuiGrid-justify-xs-space-evenly-143"
    >
      <div
        className="MuiGrid-item-123 MuiGrid-grid-xs-true-150"
      >
        <h2
          className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextPrimary-82 MuiTypography-gutterBottom-77"
        >
          choose_language
        </h2>
        <div
          className="MuiInputBase-root-301 MuiInput-root-288 MuiInput-underline-292"
          onClick={[Function]}
        >
          <div
            className="MuiSelect-root-281"
          >
            <div
              aria-haspopup="true"
              aria-owns={null}
              aria-pressed="false"
              className="MuiSelect-select-282 MuiSelect-selectMenu-285 MuiInputBase-input-311 MuiInput-input-296"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex={0}
            >
              language_en
            </div>
            <input
              type="hidden"
              value="en"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root-229 MuiSelect-icon-287"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-item-123 MuiGrid-grid-xs-true-150"
      >
        <h2
          className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextPrimary-82 MuiTypography-gutterBottom-77"
        >
          Company
        </h2>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Team
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          History
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Contact us
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Locations
        </h3>
      </div>
      <div
        className="MuiGrid-item-123 MuiGrid-grid-xs-true-150"
      >
        <h2
          className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextPrimary-82 MuiTypography-gutterBottom-77"
        >
          Features
        </h2>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Cool stuff
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Random feature
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Team feature
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Developer stuff
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Another one
        </h3>
      </div>
      <div
        className="MuiGrid-item-123 MuiGrid-grid-xs-true-150"
      >
        <h2
          className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextPrimary-82 MuiTypography-gutterBottom-77"
        >
          Resources
        </h2>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Resource
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Resource name
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Another resource
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Final resource
        </h3>
      </div>
      <div
        className="MuiGrid-item-123 MuiGrid-grid-xs-true-150"
      >
        <h2
          className="MuiTypography-root-59 MuiTypography-title-65 MuiTypography-colorTextPrimary-82 MuiTypography-gutterBottom-77"
        >
          Legal
        </h2>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Privacy policy
        </h3>
        <h3
          className="MuiTypography-root-59 MuiTypography-subheading-66 MuiTypography-colorTextSecondary-83"
        >
          Terms of use
        </h3>
      </div>
    </div>
  </footer>,
]
`;
